README.Rmd

```diff
@@ -29,7 +29,7 @@ https://www.espn.com/nba/stats/player/_/season/2019/seasontype/2
 `nba_scraper`
   
   * Scrapes data from ESPN NBA data into a csv file. User can specify the year of the season
-  (2016, 2017, etc) and the season type (regular or playoffs).
+  (2001 to 2019 at the moment) and the season type ("regular" or "playoffs").
```

Comment:
```suggestion
  (2001 to 2019 at the moment) and the season type ("regular" or "postseason").
```

Reply:
this has been changed accordingly with a more recent commit!